Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -730,10 +730,25 @@ namespace AZ
 
             m_debugCtx.ResetCullStats();
             m_debugCtx.m_numCullablesInScene = GetNumCullables();
+            AZ::JobCompletion* beginCullingCompletion = aznew AZ::JobCompletion();
```

Comment:
Nitpick: Since you're waiting for completion in the same function, you can create a JobCompletion on the stack instead of using aznew, then you don't need to check for null or delete the completion below.

Reply:
Good point, I'll change this.